scripts/mc_listen_to_corr_logger.py

```diff
@@ -0,0 +1,115 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2018 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Gather correlator log outputs and log them into M&C."""
+from __future__ import absolute_import, division, print_function
+
+import json
+import redis
+import socket
+import logging
+
+from astropy.time import Time
+
+from hera_mc import mc
+from hera_mc.correlator import DEFAULT_REDIS_ADDRESS
+
+
+allowed_levels = ["DEBUG", "INFO", "NOTIFY", "WARNING", "ERROR", "CRITICAL"]
+logging.addLevelName(logging.INFO + 1, "NOTIFY")
+
+parser = mc.get_mc_argument_parser()
+parser.add_argument(
+    "--redishost",
+    "-r",
+    dest="redishost",
+    default=DEFAULT_REDIS_ADDRESS,
+    help="The redis db hostname",
+)
+
+parser.add_argument(
+    "--channel",
+    "-c",
+    dest="channel",
+    default="mc-log-channel",
+    help="The redis channel to listen on.",
+)
+
+parser.add_argument(
+    "-l",
+    dest="level",
+    type=str,
+    default="NOTIFY",
+    help=(
+        "Don't log messages below this level. "
+        "Allowed values are {vals:}".format(vals=allowed_levels)
+    ),
+    choices=allowed_levels,
+)
+
+args = parser.parse_args()
+db = mc.connect_to_mc_db(args)
+
+hostname = socket.gethostname()
+redis_pool = redis.ConnectionPool(host=args.redishost)
+
+if args.level not in allowed_levels:
+    print("Selected log level not allowed. Allowed levels are:", allowed_levels)
+    print("Defaulting to WARNING")
+    level = logging.WARNING
```

Comment:
why are you choosing a different default here than the argument default?

Reply:
Honestly this was copy-pasta, and now that you mention it, makes little sense. I'll update to the argument default.

Reply:
actually this should be impossible to hit. I've update the parser to use the "choices" kwarg which restricts users from sending in a level that is not in "allowed_levels"

Reply:
great.